OWNERS.md

```diff
@@ -0,0 +1,12 @@
+* See [CONTRIBUTING.md](CONTRIBUTING.md) for general contribution guidelines.
+
+This page lists all active maintainers and their areas of expertise. This can be used for
+routing PRs, questions, etc. to the right place.
+
+# Maintainers
+
+* Otto van der Schaaf ([oschaaf](https://github.com/oschaaf)) (oschaaf@we-amp.com)
+  * All things Nighthawk
+* Harvey Tuch ([htuch](https://github.com/htuch)) (htuch@google.com)
```

Comment:
You can list me as "Envoy internals, architecture and performance methodology." (at least those are the things I care about, I won't claim too much expertise in any one ;). I think @jplevyak should also have "Envoy internals and architecture" listed, as he can also provide some review skills here.